src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/LanguageServer/RazorSemanticTokensScrollingBenchmark.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+#nullable disable
```

Comment:
boo :)

Reply:
I also boo this. I tried to remove it because I love me some nullable enable, but our Benchmark repo isn't laid out to make such a thing particularly easy yet.